### DISCUSSION TITLE
#30 togETHer meeting - Jan 10 2022

### DISCUSSION THREAD

**Discussion by bobjiang** [tog-ETH-er/togETHer#4] [General]
here is the agenda:

- 主题讨论 1：第三任轮值主席
- 主题讨论2：togETHer 2022展望
- 同步：各社区进展
- 自由讨论 & Teahour

**Comment by 0x-stan** [comment]
Dapp-Learning 本周工作进展：

- MIT区块链公开课翻译第七集
  - https://mp.weixin.qq.com/s/gS_s5Uy57xX5xGEBC815Zw
- 分享会
  - 《Cross-L2 Bridge: Hashed Timelock Contract vs Delayed-Proved Timelock Contract》by QiZhou
    - p1 https://www.youtube.com/watch?v=akw2Am68Fu0&t=4s
    - p2 https://www.youtube.com/watch?v=boxKV_W_OBs
    - Dr. zhou 目前已经实现了合约的demo，正在寻找志同道合的小伙伴完成项目的前端
- 《Liquity小组:小组内部讨论》by BugWriter

**Comment by winkybupt** [comment]
原语里弄：
- 邀请 ETH Denver 同伴，汇总大家关注的话题
-  陈东翻译ECDSA签名的文章，希望有社区帮忙推广
- 周四晚上万向蜂巢研习社《构建更好的Web3》邀请大家参加

**Reply by bobjiang** [reply]
陈东的那篇刚刚拜读了 - https://mp.weixin.qq.com/s/QQcIeGhe-A86BX1dfEiGKw
有意思

**Reply by yanyanho** [reply]
顶，推广起来。

**Comment by esther-ecn** [comment]
ECN：

- 翻译工作及以太坊年终总结 (L2/网络升级)
- DCL春节活动

**Comment by Riely007** [comment]
以太坊翻译小组：

- [EIP 翻译](https://eips.ethlibrary.io/zh)

自由讨论：

- 主要记录和讨论场所是 Github 的话，是否需要完善 togETHer 的 Github 介绍？

**Reply by bobjiang** [reply]
从google doc 摘取介绍放到repo readme.

**Comment by zengmi2140** [comment]
Ethfans：
* 整理 2021 年文章集合：https://ethfans.org/posts/ethfans-posts-history-2021

**Comment by Jackieyewang** [comment]
BTCU：
- 加入gen-z dao，设计dao架构和运营模式。
- 组织内容小组，规划输出内容。

**Comment by luckyyang** [comment]
Rebase
- 常规栏目 Web3 Daily、招聘信息整理和发布
- Rostra: 公会和 NFT 合约

**Comment by bobjiang** [comment]
HiBlock/Gitcoin:

- Gitcoin Q1 roadmap
- Gitcoin User Support & knowledge base

**Comment by bobjiang** [comment]
Action: 每次会议前，会议后，发邮件通知

**Reply by Riely007** [reply]
可以在轮值主席指责的文档里明确下说明下？这样之后大家都不会忘记。😄

**Comment by bobjiang** [comment]
- 以太坊meetup 增加频率，社区合作（联动）
- 1-2个社区的合作也挺好
- 黑客松
- togETHer -> DAO
- twitter space聊天 +1 +1 (yewei, Harry)
- 线下每年轮回
- 国际化（中国开发者与国际接轨，交流）
- 英文研究小组
- web3 learning 活动，包含nft, eth, 
- 品牌建设 +1 ，周边，文化衫
- 线下活动优先
- 针对（热点）主题讨论（或请人分享）
- 更多的触达，吸引更多人的可能性
- 品牌
- 组织边界（togETHer）
- 稳定的框架和结构。

**Comment by Emilyxiannv** [comment]
DAOSquare:
 Defi3.0分
[DeFi123.pdf](https://github.com/tog-ETH-er/togETHer/files/7839286/DeFi123.pdf)
享
